### PR TITLE
NuGet 6.7 Release Notes

### PR DESCRIPTION
**Pull request by jeffkl**
Related to https://github.com/NuGet/Client.Engineering/issues/2371

### CODE REVIEW

File: docs/release-notes/NuGet-6.7.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.7**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.7](https://visualstudio.microsoft.com/downloads/) | [7.0.400](https://dotnet.microsoft.com/download/dotnet-core/7.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+## Summary: What's New in 6.7
+
+* NuGetAudit should warn when no vulnerability data available - [#12610](https://github.com/NuGet/Home/issues/12610)
```
Comment (zivkan) [inline docs/release-notes/NuGet-6.7.md:22]: NuGetAudit was disabled in 6.7

File: docs/release-notes/NuGet-6.7.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+* remove period from NU190X - [#12600](https://github.com/NuGet/Home/issues/12600)
+
+* Package Source Mapping status for selected package in Details Pane - [#12586](https://github.com/NuGet/Home/issues/12586)
+
+* Add VulnerabilityInfo APIs into NuGet.Protocol - [#12518](https://github.com/NuGet/Home/issues/12518)
+
+* Signing:  raise actionable message on Linux if verification results in untrusted failure - [#12459](https://github.com/NuGet/Home/issues/12459)
+
+* Warn when vulnerabilities are detected during PackageReference restore  - [#12289](https://github.com/NuGet/Home/issues/12289)
```
Comment (zivkan) [inline docs/release-notes/NuGet-6.7.md:32]: NuGet Audit was disabled in 6.7

File: docs/release-notes/NuGet-6.7.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+* Package Source Mapping UI - PM UI Actions support source pinning & create mappings in nuget.config - [#11797](https://github.com/NuGet/Home/issues/11797)
+
+* [Feature]: Show which package versions are vulnerable in the VS PMUI package details pane version dropdown - [#11127](https://github.com/NuGet/Home/issues/11127)
+
+* Suggestion:  NuGet config -list command to list configuration file locations - [#8420](https://github.com/NuGet/Home/issues/8420)
```
Comment (zivkan) [inline docs/release-notes/NuGet-6.7.md:38]: Firstly, I believe it was only added to `dotnet nuget ...`, not to `nuget.exe`, so the name needs to be improved.

Secondly, wasn't this merged into a feature branch, but never merged into dev, and therefore isn't part of the release?
https://github.com/NuGet/NuGet.Client/tree/dev-v-dainiza-feature-dotnet-nuget-config/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands
https://github.com/NuGet/NuGet.Client/tree/release-6.7.x/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands

File: docs/release-notes/NuGet-6.7.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+**DCRs:**
+
+* There are no visual indicators for the Package Source Mapping status in details pane - [#12609](https://github.com/NuGet/Home/issues/12609)
+
+* VS Options shortcut from PMUI for PackageSourceMappings doesn't scroll to or select the Package - [#12608](https://github.com/NuGet/Home/issues/12608)
+
+* Install/Update buttons are enabled in Details Pane when the PackageSourceMapping is not found - [#12607](https://github.com/NuGet/Home/issues/12607)
+
+* NuGetAudit should check direct PackageReferences by default - [#12590](https://github.com/NuGet/Home/issues/12590)
```
Comment (zivkan) [inline docs/release-notes/NuGet-6.7.md:50]: NuGetAudit was disabled in 6.7

File: docs/release-notes/NuGet-6.7.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+
+* Add nullable annotations to NuGet.Frameworks - [#12570](https://github.com/NuGet/Home/issues/12570)
+
+* NuGetAudit should be on by default with the .NET 8 SDK - [#12568](https://github.com/NuGet/Home/issues/12568)
```
Comment (zivkan) [inline docs/release-notes/NuGet-6.7.md:54]: NuGetAudit was disabled in 6.7

File: docs/release-notes/NuGet-6.7.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+|:---|:---|:---|
+| [**6.7**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.7](https://visualstudio.microsoft.com/downloads/) | [7.0.400](https://dotnet.microsoft.com/download/dotnet-core/7.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+## Summary: What's New in 6.7
+
+* NuGetAudit should warn when no vulnerability data available - [#12610](https://github.com/NuGet/Home/issues/12610)
+
+* remove period from NU190X - [#12600](https://github.com/NuGet/Home/issues/12600)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.7.md:24]: This should probably be removed as well.

File: docs/release-notes/NuGet-6.7.md (hunk 2 above)
Comment (nkolev92) [reply docs/release-notes/NuGet-6.7.md:32]: @zivkan Should 6.8 be set as the milestone on that issue?

File: docs/release-notes/NuGet-6.7.md [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+
+* Package Source Mapping status for selected package in Details Pane - [#12586](https://github.com/NuGet/Home/issues/12586)
+
+* Add VulnerabilityInfo APIs into NuGet.Protocol - [#12518](https://github.com/NuGet/Home/issues/12518)
+
+* Signing:  raise actionable message on Linux if verification results in untrusted failure - [#12459](https://github.com/NuGet/Home/issues/12459)
+
+* Warn when vulnerabilities are detected during PackageReference restore  - [#12289](https://github.com/NuGet/Home/issues/12289)
+
+* Package Source Mapping UI - PM UI Actions support source pinning & create mappings in nuget.config - [#11797](https://github.com/NuGet/Home/issues/11797)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.7.md:34]: @donnie-msft 
Should this be moved to changes not in the main new features section? This is largely a technically change that's not enabled for users just yet as the PM UI doesn't call this codepath yet.

File: docs/release-notes/NuGet-6.7.md (hunk 3 above)
Comment (nkolev92) [reply docs/release-notes/NuGet-6.7.md:38]: I agree. We need to remove this. I've changed the milestone for that.

**Review by nkolev92 — COMMENTED** [review]
Looks pretty good. Just a few small comments.

File: docs/release-notes/NuGet-6.7.md (hunk 1 above)
Comment (jeffkl) [reply docs/release-notes/NuGet-6.7.md]: Good point, I've updated the release notes for all of the NuGetAudit stuff

File: docs/release-notes/NuGet-6.7.md (hunk 7 above)
Comment (donnie-msft) [reply docs/release-notes/NuGet-6.7.md:28]: Yep!

**Comment by jeffkl** [comment]
@JonDouglas any comments?

**Review by JonDouglas — APPROVED** [review]
Looks good to me, please review the blog post to ensure all features represented are available.

File: docs/release-notes/NuGet-6.7.md (hunk 7 above)
Comment (nkolev92) [reply docs/release-notes/NuGet-6.7.md:28]: Want to make sure we don't forget about this.

**Comment by jeffkl** [comment]
Team Review: Re-run the automated release notes tool

File: docs/release-notes/NuGet-6.7.md (hunk 7 above)
Comment (jeffkl) [reply docs/release-notes/NuGet-6.7.md:28]: I'll just remove this for now?  Or should I put it under Bugs/DCRs?

Comment (nkolev92) [reply docs/release-notes/NuGet-6.7.md:28]: I'd just remove it for now.

Comment (jeffkl) [reply docs/release-notes/NuGet-6.7.md]: Removed

**Pull request merged**